browser/css/notebookbar.css

```diff
@@ -337,15 +335,6 @@
 	margin-top: -5px;
 }
 
-#clearFormatting.notebookbar div img, #FormatPaintbrush.notebookbar div img {
-	width: 24px !important;
-	height: 24px !important;
-}
-
-#clearFormatting.notebookbar, #FormatPaintbrush.notebookbar{
-	height: 24px !important;
-}
-
```

Comment:
Seems to be unrelated to this commit. Could we please have this in a different commit?